stories/molecules/components/footer/Footer.stories.mdx

```diff
@@ -0,0 +1,189 @@
+import { ArgsTable, Meta, Story, Canvas, Preview, Props } from '@storybook/addon-docs'
+import { ThemeProvider } from 'ui/atoms/theme/ThemeProvider'
+import { ThemeSwitcher } from 'ui/atoms/theme/ThemeSwitcher'
+import { Footer } from 'ui/molecules/footer/Footer'
+import { Typography } from 'ui/atoms/typography/Typography'
+import { loadTranslations } from 'i18n/utils'
+import { useTranslation } from 'hook/useTranslation'
+import footer_en from './i18n/footer_en.json'
+import footer_fr from './i18n/footer_fr.json'
+import footer_se from './i18n/footer_se.json'
+import footer_it from './i18n/footer_it.json'
+import { Button } from 'ui/atoms/button/Button'
+
+<Meta
+  title="Molecules/Footer"
+  component={Footer}
+  parameters={{
+    actions: { argTypesRegex: '^on.*' },
+    docs: {
+      source: {
+        type: 'code',
+        format: true,
+        language: 'jsx'
+      }
+    }
+  }}
+  decorators={[
+    Story => {
+      return (
+        <ThemeProvider>
+          <div style={{ padding: '20px 30px' }}>
+            <ThemeSwitcher />
+          </div>
+          <Story />
+        </ThemeProvider>
+      )
+    }
+  ]}
+/>
+
+# Footer
+
+> Helps users to find useful information and site functionality.
```

Comment:
Maybe we can turn this sentence into plural, it's up to you.
```suggestion
> Helps users to find useful informations and site functionalities.
```

Reply:
Yes makes sense, although information is a mass noun in english and as such the spelling does not change, see [This explanation](https://english.stackexchange.com/questions/117552/why-does-information-not-have-a-plural-form).